crates/tabby-scheduler/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+//! Responsible for scheduling all of the background tasks for tabby.
```

Comment:
```suggestion
//! Responsible for scheduling all of the background jobs for tabby.
```

used `jobs` since we have subcommand concept `job::` for backend jobs.